### ISSUE TITLE
Failure in #1423 is due AnnexRepo.get() leaking progress info in return JSON

### ISSUE THREAD

**Issue by mih** [datalad/datalad#1440]
Result of getting a single file looks like this in PY3:

````
[{'total-size': 13702811, 'percent-progress': '13.84%', 'byte-progress': 1896448, 'action': {'file': 'sub-01/anat/sub-01_T1w.nii.gz', 'note': 'from mddatasrc...', 'key': 'MD5E-s13702811--21c3c396e96b308b021b5754afa5518c.nii.gz', 'command': 'get'}},
{'total-size': 13702811, 'percent-progress': '36.47%', 'byte-progress': 4997120, 'action': {'file': 'sub-01/anat/sub-01_T1w.nii.gz', 'note': 'from mddatasrc...', 'key': 'MD5E-s13702811--21c3c396e96b308b021b5754afa5518c.nii.gz', 'command': 'get'}},
{'total-size': 13702811, 'percent-progress': '59.04%', 'byte-progress': 8089600, 'action': {'file': 'sub-01/anat/sub-01_T1w.nii.gz', 'note': 'from mddatasrc...', 'key': 'MD5E-s13702811--21c3c396e96b308b021b5754afa5518c.nii.gz', 'command': 'get'}},
{'total-size': 13702811, 'percent-progress': '81.63%', 'byte-progress': 11186176, 'action': {'file': 'sub-01/anat/sub-01_T1w.nii.gz', 'note': 'from mddatasrc...', 'key': 'MD5E-s13702811--21c3c396e96b308b021b5754afa5518c.nii.gz', 'command': 'get'}},
 {'file': 'sub-01/anat/sub-01_T1w.nii.gz', 'success': True, 'note': 'checksum...', 'key': 'MD5E-s13702811--21c3c396e96b308b021b5754afa5518c.nii.gz', 'command': 'get'}]
````

The only actual result is the last item.

**Comment by yarikoptic** [comment]
would have been useful to know the version of datalad/annex since can't reproduce
```shell
$> python --version
Python 3.5.3
(python3) 2 4998.....................................:Sun 26 Mar 2017 12:28:10 PM EDT:.
hopa:~/datalad/openfmri/ds000001
$> python -c 'from datalad.support.annexrepo import AnnexRepo; AnnexRepo(".").get("sub-01/anat/sub-01_T1w.nii.gz")'
(python3) 2 4999.....................................:Sun 26 Mar 2017 12:29:29 PM EDT:.
hopa:~/datalad/openfmri/ds000001
$> git annex version
git-annex version: 6.20170307+gitg24ade8a25-1~ndall+1
...
(python3) 2 5001.....................................:Sun 26 Mar 2017 12:31:16 PM EDT:.
hopa:~/datalad/openfmri/ds000001
$> datalad --version
datalad 0.5.1.dev1
```
and there were progressbars as they should and all is clean (

**Comment by mih** [comment]
Travis showed the same behavior for previous commits in this PR. Just check
the logs, and same version as travis.

**Comment by yarikoptic** [comment]
this is an issue not a  PR ;-P  if you are talking about the 1423 -- damn bus connection is too slow to jump around github/travis but IIRC it as merged and I felt like it was merged before I have fixed up some stuff for python3 in 0.5.1.   so again -- confirmation that it is present on recenty released datalad and recent git-annex would be helpful.

**Comment by mih** [comment]
It happened in this PR only, and only in the example run as part of a datalad get call. Python 3.5.3, git-annex 6.20170220+gitg75a15e1ad-1~ndall+1. I have never seen this before or after, but I could reliably reproduce the travis error on my machine.

**Comment by yarikoptic** [comment]
was fixed in  	8c6259e
